integration-test/src/main/resources/testsuites/clustertestset.yaml

```diff
@@ -2,8 +2,8 @@ name: Cluster_tests
 parallel: tests
 threadCount: 4
 parameters:
-  awsRegion: eu-west-1
```

Comment:
I think this file is not used anymore. Instead edit the application.yml. Please correct me if I'm wrong.